pkg/server/router/keystore.go

```diff
@@ -37,14 +37,14 @@ type StoreKeyRequest struct {
 	ID string `json:"id" validate:"required"`
 
 	// Identifies the cryptographic algorithm family used with the key.
-	// One of the following: `"Ed25519","X25519","secp256k1","P-224","P-256","P-384","P-521","RSA"`.
+	// One of the following: "Ed25519", "X25519", "secp256k1", "P-224", "P-256", "P-384", "P-521", "RSA".
 	Type crypto.KeyType `json:"type,omitempty" validate:"required"`
 
 	// See https://www.w3.org/TR/did-core/#did-controller
 	Controller string `json:"controller,omitempty" validate:"required"`
 
 	// Base58 encoding of the bytes that result from marshalling the private key using golang's implementation.
-	PrivateKeyBase58 string `json:"base58PrivateKey,omitempty" validate:"required"`
+	PrivateKeyBase58 string `json:"base58PrivateKey,omitempty"`
```

Comment:
remove validate:"required" was this intendend ?

Reply:
I can add this back until #371